Cargo.toml

```diff
@@ -17,7 +17,9 @@ version = "1.0.0-alpha.4" # remember to update html_root_url
 
 [features]
 # Enabling this feature enables the `futures` module using generic associated types (GATs), which are still unstable.
-unstable-gats = []
+# Therefore, this feature requires compiling on nightly.
+futures = []
```

Comment:
I think we do need the word "unstable" somewhere, signaling that "it might get breaking changes even in minor releases"